nengo/simulator.py

```diff
@@ -55,11 +55,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    # 'unsupported' defines features unsupported by a simulator.
+    # The format is a list of tuples of the form `(test, reason)` with `test`
+    # being a regular expression matched against Nengo test names, and `reason`
+    # is a string describing why the feature is not supported by the backend.
+    # E.g., unsupported = [('test_pes*', 'PES rule not implemented')]
```

Comment:
I don't think this is a good example. Remember this is regex. So isn't this going to look for "test_pe" with 0 or more esses at the end?